### PR TITLE
add logic

### PR DESCRIPTION
**Pull request by alysonz**

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var query = require('pg-query');
```
Comment (phated) [inline client.js]: move this whole file to the index.js, no need to have it separate

**Comment by alysonz** [comment]
updated

**Comment by phated** [comment]
@alysonz it looks like a dependency for `pg.js` must also be added because pg-query doesn't have a dependency on it but it is required (they are kinda doing a peer dependency thing without peerDependencies in package.json).  I ran into this by linking the module into our migrations.

**Comment by alysonz** [comment]
updated

**Pull request merged**